x509-cert/Cargo.toml

```
@@ -23,6 +23,9 @@ spki = { version = "0.7.2", features = ["alloc"] }
 arbitrary = { version = "1.3", features = ["derive"], optional = true }
 sha1 = { version = "0.10.0", optional = true }
 signature = { version = "2.1.0", features = ["rand_core"], optional = true }
+tls_codec = { version = "0.4.0", default-features = false, features = [
+    "derive",
+], optional = true }
```

We usually do these as one-liners...

```suggestion
tls_codec = { version = "0.4.0", default-features = false, features = ["derive"], optional = true }
```